addons/sale/report/sale_report.py

```diff
@@ -236,3 +242,12 @@ def _query(self):
     @property
     def _table_query(self):
         return self._query()
+
+    def action_open_order(self):
+        self.ensure_one()
+        return {
+            'res_model': self.order_reference._name,
+            'type': 'ir.actions.act_window',
+            'views': [[False, "form"]],
```

Comment:
```suggestion
            'views': [[False, 'form']],
```
single quoted tech string

Reply:
Done